.bazelrc_shared

```diff
@@ -22,14 +22,3 @@ test --test_output=all
 
 build:rules --disk_cache=.bazel_cache
 build:tests --disk_cache=../.bazel_cache
-
-common:rules_v7.2 --config=noop
```

Comment:
Do you know what this deleted portion was doing?

Reply:
The per version config existed so we could set flags for certain versions of Bazel and not others. Back before 1.0 things would change often enough that we'd often need a different set of flags for each version.

Now that we're really only supporting 1 version of Bazel (because we only use these rules internally) we haven't had to do that for a while.

The noop config was a way to have a version config that did nothing. The version configs were automatically generated by the old `tools/bazel`, so you had to have them point to something, even if they did nothing.